app/helpers/operating_systems_helper.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module OperatingSystemsHelper
+  def operating_system_options
```

Comment:
@Kenneth-Schroeder @regnujAx @hpi-swt2/bd-18 I dont like this any better ideas (how would you do this in an other language)? 
Basically I just want to get all OS names and "none" as the first array entry and "other" as the last one. (note: in ruby last line is returned without explicit return)

Reply:
is there some equivalent to push_front?

Reply:
@Kenneth-Schroeder yea i thought about this as well, would be one more line but also clearer...

Reply:
couldn't you just do something like:

options = OperatingSystem.all.each
options.pushfront 'none'
options.pushback 'other'

(saving the loop)
?

Reply:
https://stackoverflow.com/questions/8131669/populating-an-array-from-another-array

> ```
> x = ["abcd", "efgh", "able"]
> y = x.map{|e| e[0]}          # keeps x intact
> ```

Reply:
tyty :*